src/patterns/atoms/block/paragraph/paragraph.stories.js

```diff
@@ -0,0 +1,14 @@
+import paragraph from './paragraph.twig';
+
+export default {
+  title: 'Atoms|Block/Paragraph',
+};
+
+export const Basic = () => paragraph({
```

Comment:
Object should be on a new line to match others. Is there an eslint rule for this?

Reply:
I thought there would be, but can't find it. Might be able to combine rules to give us what we want... Will have a play, but don't want to hold up getting this in.